src/OpenTelemetry.Contrib.Instrumentation.Remoting/README.md

```diff
@@ -0,0 +1,124 @@
+# .NET Remoting Instrumentation for OpenTelemetry.Contrib .NET
+
+This is an instrumentation library, which instruments [.NET Remoting](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-3.0/72x4h507(v=vs.85))
```

Comment:
Can you remove "en-us/" part from docs link, so it'll always redirect based on user's settings.

Reply:
Done!

Reply:
Can you check this for this entire PR . there are a lot of links which has en-us in it!
(we may be able to automatically flag this in CI)

Reply:
Thanks, missed the ones in the code itself, updated now. I see there are two more in the Common msbuild props, won't touch them as part of this PR but can do a tiny separate one to clean those up as well.